### ISSUE TITLE
[BUG] ubi-micro image doesn't contain sed, sidecar loggers fail to output logs

### ISSUE THREAD

**Issue by gangefors** [jfrog/charts#1724]
**BUG REPORT**

**Which chart**:
jfrog-platform 10.11.1

**What happened**:
```
kubectl -n jfrog logs jfrog-platform-artifactory-2 artifactory-service-log
/scripts/tail-log.sh: line 14: sed: command not found
```

**What you expected to happen**:
```
kubectl -n jfrog logs jfrog-platform-artifactory-2 artifactory-service-log
Log file ./artifactory-service.log is ready!
```

**How to reproduce it** (as minimally and precisely as possible):
Deploy jfrog-platform chart v10.11.1 with sidecar loggers enabled

**Solution**
Change logging sidecar image to 
```
artifactory:
  logger:
    image:
      registry: releases-docker.jfrog.io
      repository: ubi8/ubi-minimal
      tag: 8.7.1049
```

**Comment by srinivasgowda097** [comment]
@gangefors Thanks for reporting it, It will be fixed in upcoming releases

**Comment by chukka** [comment]
Fixed in #1728
